Specs/Scene/ExpressionSpec.js

```diff
@@ -1807,6 +1879,34 @@ defineSuite([
         expect(shaderExpression).toEqual(expected);
     });
 
+    it('gets shader expression for atan2', function() {
+        var expression = new Expression('atan2(0,1)');
+        var shaderExpression = expression.getShaderExpression('', {});
+        var expected = 'atan2(0,1)';
+        expect(shaderExpression).toEqual(expected);
+    });
+
+    it('gets shader expression for pow', function() {
+        var expression = new Expression('pow(2,2)');
+        var shaderExpression = expression.getShaderExpression('', {});
+        var expected = 'pow(2,2)';
+        expect(shaderExpression).toEqual(expected);
+    });
+
+    it('gets shader expression for min', function() {
+        var expression = new Expression('min(3,5)');
+        var shaderExpression = expression.getShaderExpression('', {});
+        var expected = 'min(3,5)';
+        expect(shaderExpression).toEqual(expected);
+    });
+
+    it('gets shader expression for max', function() {
+        var expression = new Expression('max(3,5)');
+        var shaderExpression = expression.getShaderExpression('', {});
+        var expected = 'max(3,5)';
```

Comment:
All of these expected lines need to be like `max(3.0, 5.0)`. This came up in @Dylan-Brown's PR too: https://github.com/AnalyticalGraphicsInc/cesium/pull/4578#issuecomment-258256974